### PR TITLE
Pot/Potluck Report

### PR DESCRIPTION
**Pull request by hny-gd**

### CODE REVIEW

File: website/content/en/status/report-2023-07-2023-09/pot.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@FreeBSD.org> +
+Contact: Bretton Vine (Potluck) <bv@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@FreeBSD.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During this quarter, link:https://github.com/bsdpot/pot/pull/274[Pot 0.15.6] was finished, adding custom pf rule configuration hooks.
+
+Additionally, a new release for the link:https://github.com/bsdpot/nomad-pot-driver[Nomad Pot Driver] that allows setting Pot attributes in Nomad job descriptions will very likely be available by the time you are reading this report.
```
Comment (grembo) [inline website/content/en/status/report-2023-07-2023-09/pot.adoc:14]: @hny-gd v0.9.1 was released today, portstree update will happen tomorrow latest.

File: website/content/en/status/report-2023-07-2023-09/pot.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+The July/August 2023 edition of the FreeBSD Journal contains Luca's link:https://freebsdfoundation.org/wp-content/uploads/2023/08/Pizzamiglio.pdf[Jail Orchestration with pot and nomad] article, explaining how to use Pot and Potluck together with Nomad to orchestrate containers on multiple hosts.
+
+Last not least, link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=273583[a patch] for the link:https://www.freshports.org/devel/sccache/[sccache port] was submitted to include sccache-dist and is about to be released at the time of this writing.
```
Comment (grembo) [inline website/content/en/status/report-2023-07-2023-09/pot.adoc:22]: @hny-gd This also landed today.

**Review by grahamperrin — CHANGES_REQUESTED** [review]

File: website/content/en/status/report-2023-07-2023-09/pot.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Links: +
+link:https://github.com/bsdpot[Pot organization on GitHub] URL: link:https://github.com/bsdpot[]
+
+Contact: Luca Pizzamiglio (Pot) <pizzamig@FreeBSD.org> +
+Contact: Bretton Vine (Potluck) <bv@honeyguide.eu> +
+Contact: Michael Gmelin (Potman) <grembo@FreeBSD.org>
+
+Pot is a jail management tool that link:https://www.freebsd.org/news/status/report-2020-01-2020-03/#pot-and-the-nomad-pot-driver[also supports orchestration through Nomad].
+
+During this quarter, link:https://github.com/bsdpot/pot/pull/274[Pot 0.15.6] was finished, adding custom pf rule configuration hooks.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-07-2023-09/pot.adoc:12]:
```suggestion
During this quarter, link:https://github.com/bsdpot/pot/pull/274[Pot 0.15.6] was finished, adding custom man:pf[4] rule configuration hooks.
```

File: website/content/en/status/report-2023-07-2023-09/pot.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Quite a few new container images were made available, e.g. link:https://github.com/bsdpot/potluck/tree/master/caddy-s3-nomad[a Caddy S3 proxy], link:https://github.com/bsdpot/potluck/tree/master/mastodon-s3[a Mastodon instance] or a link:https://github.com/bsdpot/potluck/tree/master/redis-single[Redis container]. In total there are now 50 containers available that can either be downloaded as ready-made images at link:https://potluck.honeyguide.net/[the Potluck image registry] if you trust our build process or that you can rebuild yourself from the Pot flavour files stored in the link:https://github.com/bsdpot/potluck[Potluck Github repository].  
+
+The July/August 2023 edition of the FreeBSD Journal contains Luca's link:https://freebsdfoundation.org/wp-content/uploads/2023/08/Pizzamiglio.pdf[Jail Orchestration with pot and nomad] article, explaining how to use Pot and Potluck together with Nomad to orchestrate containers on multiple hosts.
+
+Last not least, link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=273583[a patch] adding build cluster support to the link:https://www.freshports.org/devel/sccache/[sccache port] was committed.
```
[the remark this answers is not in the document]
Comment (lsalvadore) [reply website/content/en/status/report-2023-07-2023-09/pot.adoc:22]: I think the commit is in `ports` not in `src`, hence
```suggestion
Last but not least, a patch (gitref:90b1184d93c8[repository=ports]) added build cluster support to the package:devel/sccache[] port.
```

**Review by lsalvadore — CHANGES_REQUESTED** [review]

**Comment by hny-gd** [comment]
@grahamperrin and @lsalvadore thank you very much for your edits, I committed them.

File: website/content/en/status/report-2023-07-2023-09/pot.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (grahamperrin) [reply website/content/en/status/report-2023-07-2023-09/pot.adoc]: @lsalvadore thanks, sorry for my error (copy-and-paste from Bugzilla, an oversight with edition).

**Review by lsalvadore — APPROVED** [review]
Approved, thanks.

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**